bin/run-cabal

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -xe
+export PORT=9001
+
+# This will run ghcid, which uses `./.ghcid` to invoke your program main entry
+# point, with the specified args.
+#
+# If you change ./.ghcid, ghcid will automatically reload.
+ghcid --command "cabal repl"
```

Comment:
Is `--command "cabal repl"` required? Doesn't running just `ghcid` (ie. without arguments) work?

Because if it did, we can make `bin/run` just call this script.

Reply:
Running ghcid with arguments invokes `stack repl` by default AFAUI.